pkg/lock/simple_redis.go

```diff
@@ -64,45 +51,97 @@ type redisLock struct {
 	mu     sync.Mutex
 	key    string
 	token  string
```

Comment:
Lint: I think there is one space missing before type here to align with the 3 lines below following the comment

Reply:
Well, you know, go developers don't discuss things like that. They use `gofmt`, and `gofmt` is always right (or, in the extreme case where `gofmt` is wrong, we should open a issue for it, not discuss it here).

`gofmt` does not try to align types before and after a comment, and as said before, `gofmt` is always right. So, let's keep it as is.

Reply:
👍